src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingMethodSymbol.cs

```diff
@@ -280,6 +280,16 @@ private ImmutableArray<MethodSymbol> RetargetExplicitInterfaceImplementations()
             return builder.ToImmutableAndFree();
         }
 
+        internal override MethodSymbol ExplicitlyOverriddenClassMethod
+        {
+            get
+            {
+                return _underlyingMethod.OverriddenMethod is MethodSymbol overridden
```

Comment:
>OverriddenMethod [](start = 41, length = 16)

This looks like a bug. Should this be ```ExplicitlyOverriddenClassMethod```? It looks like there is a test gap as well.

Reply:
No, this is intended and necessary for correctness. Most of the tests would break if changed as suggested. Having said that, this method is removed later in this PR.

---
In reply to: [424665540](https://github.com/dotnet/roslyn/pull/44025#discussion_r424665540) [](ancestors = 424665540)